integration_tests/handler/0_utils_test.go

```diff
@@ -11,21 +11,55 @@ import (
 	"github.com/gofrs/uuid"
 	"github.com/labstack/echo/v4"
 	"github.com/stretchr/testify/assert"
-	"github.com/traPtitech/traPortfolio/infrastructure"
+	"github.com/traPtitech/traPortfolio/infrastructure/external/mock_external_e2e"
+	"github.com/traPtitech/traPortfolio/infrastructure/repository"
 	"github.com/traPtitech/traPortfolio/integration_tests/testutils"
 	"github.com/traPtitech/traPortfolio/interfaces/handler"
+	"github.com/traPtitech/traPortfolio/util/config"
 	"github.com/traPtitech/traPortfolio/util/mockdata"
 	"github.com/traPtitech/traPortfolio/util/random"
+	"gorm.io/gorm"
 )
 
+func injectIntoAPIServer(t *testing.T, c *config.Config, db *gorm.DB) (handler.API, error) {
+	t.Helper()
+
+	// モック前提のテストがあるためassert; FIXME
```

Comment:
```suggestion
	// FIXME: モック前提のテストがあるためassert
```
にしてくれると僕の環境でFIXMEがハイライトしてくれるのでそうしてくれると助かる
https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree